scripts/globi-interaction.json

```diff
@@ -0,0 +1,233 @@
+{
+  "citation": "NA",
```

Comment:
The citation should be:

Poelen, J.H., Simons, J.D. and Mungall, C.J., 2014. Global biotic interactions: an open infrastructure to share and analyze species-interaction datasets. Ecological Informatics, 24, pp.148-159.